test/functional/hosts_controller_test.rb

```diff
@@ -311,6 +311,16 @@ def setup_user_and_host(operation, filter = nil, &block)
     end
   end
 
+  def test_multiple_rebuild_tftp
+    h = FactoryGirl.create(:host, :interfaces => [ FactoryGirl.build(:nic_primary_and_provision) ])
+    refute_nil h
+    refute_nil h.primary_interface
+
+    post :submit_rebuild_tftp, { :host_ids => [h.id] }, set_session_user
```

Comment:
It'd be helpful to have an expectation somewhere, e.g. you expect that setTFTP (or whatever) is called, or some proxy API etc.